docs/workflow_generators/residential_hpxml.rst

```diff
@@ -14,11 +14,11 @@ Configuration Example
 
         emissions:
           - scenario_name: Scenario1
-            type: CO2
+            type: CO2e
             elec_folder: data/cambium/LRMER_MidCase_15
-            gas_value: 117.6
-            propane_value: 136.6
-            oil_value: 161.0
+            gas_value: 147.3
+            propane_value: 177.8
+            oil_value: 195.9
             wood_value: 200.0
```

Comment:
Where did this wood value come from?

Reply:
It's made up.

Reply:
@ekpresent @ejhw How do you want to handle emissions for homes with wood heating?

Reply:
@shorowit I would propose zero CO2e. Is that different from how other HPXML workflows handle it? When we add criteria pollutants then it will important to include the criteria pollutants from wood burning.